app/code/community/Twispay/Tpay/Model/PaymentMethod.php

```diff
@@ -1,64 +1,69 @@
-    protected $_canUseForMultishipping  = true;
-    protected $_canSaveCc               = false;
-    protected $_isInitializeNeeded      = false;
+    protected $_formBlockType = 'tpay/form_tpay';
 
+    protected $_isGateway = true;
+    protected $_canAuthorize = true;
+    protected $_canCapture = true;
+    protected $_canCapturePartial = true;
+    protected $_canRefund = false;
+    protected $_canVoid = false;
+    protected $_canUseInternal = true;
+    protected $_canUseCheckout = true;
+    protected $_canUseForMultishipping = true;
+    protected $_canSaveCc = false;
+    protected $_isInitializeNeeded = false;
+  
     /**
-    * @return Mage_Checkout_Model_Session
-    */
-    protected function _getCheckout()
+     *
+     * @return Mage_Checkout_Model_Session
+     */
+    protected function _getCheckout() 
     {
-       return Mage::getSingleton('checkout/session');
+        return Mage::getSingleton ( 'checkout/session' );
     }
-
+    
     // Construct the redirect URL
-    public function getOrderPlaceRedirectUrl()
-    {   
-        $redirect_url = Mage::getUrl('tpay/payment/redirect');
-        Mage::Log("Step 2 Process: Getting the redirect URL: $redirect_url", Zend_Log::DEBUG, $this->LOG_FILE_NAME);
-        return $redirect_url;      
+    public function getOrderPlaceRedirectUrl() 
+    {
+        Mage::log('getOrderPlaceRedirectUrl', null, 'twispay.log', true);
+        $redirectUrl = Mage::getUrl ( 'tpay/payment/redirect' );
+        Mage::Log ( "Step 2 Process: Getting the redirect URL: $redirectUrl", Zend_Log::DEBUG, $this->logFileName );
+        return $redirectUrl;
     }
 
-    public function authorize(Varien_Object $payment, $amount){
-        Mage::Log('Step 0 Process: Authorize', Zend_Log::DEBUG, $this->LOG_FILE_NAME);
-        return $this;               
+    /**
+     * Function used to force the authorization of any received order.
+     */
+    public function authorize(Varien_Object $payment, $amount) 
+    {
+        Mage::log('authorize', null, 'twispay.log', true);
+        Mage::Log ( 'Step 0 Process: Authorize', Zend_Log::DEBUG, $this->logFileName );
+        return $this;
     }
+
     /**
      * this method is called if we are authorising AND
      * capturing a transaction
      */
-    public function capture(Varien_Object $payment, $amount)
+    public function capture(Varien_Object $payment, $amount) 
     {
-        Mage::Log('Step 1 Process: Create and capture the process', Zend_Log::DEBUG, $this->LOG_FILE_NAME);
+        Mage::log('capture', null, 'twispay.log', true);
+        Mage::Log ( 'Step 1 Process: Create and capture the process', Zend_Log::DEBUG, $this->logFileName );
         return $this;
     }
-    
-    public function serverurl()
+    public function serverurl() 
     {
-    	Mage::Log("running serverurl", Zend_Log::DEBUG, $this->LOG_FILE_NAME);
-    	return "testurlserver";
+        Mage::log('serverurl', null, 'twispay.log', true);
```

Comment:
Please remove the line `Mage::log('serverurl', null, 'twispay.log', true);`

Reply:
Removed